Changes

```diff
@@ -26,6 +26,10 @@ Revision history for Perl extension App::Sqitch
        override it. Use this variable to prevent accidental reverts in
        sensitive environments. Thanks to @vectro for the PR (#719; revised in
        #735)!
+     - Fixed test failures due to a bug fix in the Perl URI module
+       (libwww-perl/URI#13). Thanks to @bobfang for the report (#744)!
+     - Fixed test failures due to a in the generation of DBI DSN by URI::Oracle
```

Comment:
a _bug_ in?

Reply:
Bah, a _change_ in. Fixed in bcf02509, thanks.